File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    if(x<0){
        return false
    }
    else{
        let y = x.toString().split("").reverse().join("")
        if(y==x){
            return true
        }
        else{
            return false
        }
    }
};

File: 0009-palindrome-number/0009-palindrome-number.cs
public class Solution {
    public bool IsPalindrome(int x) {
        if(x < 0) return false;

        string og = x.ToString();
        char[] charArray = og.ToCharArray();
        Array.Reverse(charArray);
        string reversed = new string(charArray);

        return og == reversed;
    }
}